code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm

```diff
@@ -277,16 +277,22 @@
 			to_chat(src, "<span class='xenowarning'>We tuck ourselves into a defensive stance.</span>")
 		GLOB.round_statistics.defender_crest_lowerings++
 		SSblackbox.record_feedback("tally", "round_statistics", 1, "defender_crest_lowerings")
-		soft_armor = soft_armor.setRating(bomb = XENO_BOMB_RESIST_2)
-		armor_bonus += xeno_caste.crest_defense_armor
+		if(silent) //if we were fortifying remove fortifying bonus first
```

Comment:
this is not what silent is meant for, idk what the hell you're doing here

Reply:
silent says if he used the fortifying, so it doenst give double message, its literally that

Reply:
silent is only for not display user output, the ability should work exactly the same

Reply:
but it only gets set on when the user is fortifying, but i'll do it another way around.